docker/edge/http/script.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+#apt-get install jq -y
+
+#Read the Keyrock IP, Application name, redirect_uri and application URL from user
+echo Enter the Keyrock IDM IP:
+read IP
+IP_ADDRESS=`echo $IP | tr -d '\n'`
+echo $IP | tr -d '\n'|od -c
+echo IDM IP is $IP_ADDRESS
+echo -----------------------------
+echo Enter name of Application:
+read name
+appName=`echo $name | tr -d '\n'`
+echo Name of application is $appName
+echo -----------------------------
+echo Enter redirect_uri/Callback_uri:
+read callback
+callbackUrl=`echo $callback | tr -d '\n'`
+echo -----------------------------
+
+echo Enter application url:
+read Uri
+appUri=`echo $Uri | tr -d '\n'`
+echo -----------------------------
+
+#Generate API token, It will return X-Subject-Token taht will further use for application register
+curl --include \
+     --request POST \
+     --header "Content-Type: application/json" \
+     --data-binary "{
+  \"name\": \"admin@test.com\",
+  \"password\": \"1234\"
+}" \
+"http://$IP_ADDRESS:3000/v1/auth/tokens" > generate_token.txt
+
+token=`grep "X-Subject-Token" generate_token.txt | cut -f 2 -d ":" |  sed 's/\r$//g' | tr -d '\n'`
+#token_id=`echo $token | sed 's/ //g'`
+#echo $token | sed 's/ //g' > token.txt
+echo IDM token is $token
+echo -----------------------------
+#echo "######## 1"
+
+#Register application, it will return Application ID and Password. ID and Password will use to genenrate access token
+curl -sb --include --request POST  -H "Content-Type: application/json"  -H "X-Auth-token: $token" --data-binary "{
+  \"application\": {
+    \"name\": \"$appName\",
+    \"description\": \"description\",
+    \"redirect_uri\": \"$callbackUrl\",
+    \"url\": \"$appUri\",
+    \"grant_type\": [
+      \"authorization_code\",
+      \"implicit\",
+      \"password\"
+    ],
+    \"token_types\": [
+        \"jwt\",
+        \"permanent\"
+    ]
+  }
+}" \
+"http://$IP_ADDRESS:3000/v1/applications" > accessAPI.txt
```

Comment:
[Minor] there might be some filesystem where other user than root don't have file create permission so better to run whole script in root user mode.